loopout.go

```diff
@@ -43,6 +43,10 @@ var (
 	// TODO(wilmer): tune?
 	DefaultSweepConfTargetDelta = DefaultSweepConfTarget * 2
 
+	// DefaultMaxMinerFeePortion is the max portion of the total swap
+	// amount which can be used as miner fee when sweeping the HTLCs.
+	DefaultMaxMinerFeePortion = 0.05
```

Comment:
How did you pick this default? 

For our min/max swap amount, this would mean we have fees in the range of 12500-50000 sat.